### ISSUE TITLE
Can't check CheckBox in DataGridTemplateColumn [11preview5]

### ISSUE THREAD

**Issue by unofficialdev** [AvaloniaUI/Avalonia#10570]
**Describe the bug**
Can't check CheckBox in DataGridTemplateColumn

**Expected behavior**
check CheckBox

- OS: [e.g. Windows 10
- Version 20H2 19045.2673
**Additional context**
I use CheckBox in DataGridTemplateColumn and catch Checked event. here is the code
```
<DataGridTemplateColumn Width="35" Header="#">
	<DataGridTemplateColumn.CellTemplate>
		<DataTemplate>
			<Grid>
				<CheckBox
                                         Margin="10,0,0,0"
                                        Checked="OnStartCheck"
                                        IsChecked="{CompiledBinding IsChecked}"
                                        Unchecked="OnStartCheck"/>
			</Grid>
		</DataTemplate>
	</DataGridTemplateColumn.CellTemplate>
</DataGridTemplateColumn>
```

Everything works fine in the old version. After updating to 11preview5, the check boxes can't work

**Comment by rabbitism** [comment]
Should be fixed by https://github.com/AvaloniaUI/Avalonia/pull/10252

**Comment by timunie** [comment]
Please try nightly builds. The issue was fixed after 11.0 preview5 release.

**Comment by timunie** [comment]
Closing issue as it is fixed in nightly. If you disagree let me know and I'll reopen it.
